spec/amber/pipes/static_spec.cr

```diff
@@ -22,7 +23,7 @@ module Amber
 
         response = create_request_and_return_io(static, request)
 
-        response.body.should eq "Not Found\n"
```

Comment:
Did this change in this version of crystal?

Reply:
Yeah looks like it was changed, the most recent version of master has it with a new method (https://github.com/crystal-lang/crystal/blob/master/src/http/server/response.cr#L144) which prepends the status code before the message.